applications/tari_dan_wallet_cli/src/command/transaction.rs

```diff
@@ -566,6 +566,9 @@ pub fn summarize_finalize_result(finalize: &FinalizeResult) {
                     SubstateValue::Resource(_) => {
                         println!("      ▶ resource: {}", address);
                     },
+                    SubstateValue::TransactionReceipt(_) => {
+                        println!("      ▶ execute_result: {}", address);
```

Comment:
`println!("      ▶ transaction_receipt: {}", address);`

Reply:
This is very valuable ! Thanks a lot for catching these up!